elasticsearch/src/main/java/org/hibernate/search/elasticsearch/bridge/builtin/impl/Elasticsearch.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.elasticsearch.bridge.builtin.impl;
+
+import org.hibernate.search.bridge.spi.FieldMetadataCreationContext;
+import org.hibernate.search.bridge.spi.FieldType;
+import org.hibernate.search.elasticsearch.schema.impl.model.DynamicType;
+import org.hibernate.search.engine.metadata.impl.BridgeDefinedField;
+
+/**
+ * @author Davide D'Alto
+ */
+public class Elasticsearch implements FieldMetadataCreationContext {
```

Comment:
Sorry about that, but I just noticed: this class is part of an implementation package. If we want to make it available for use in custom field bridges, shouldn't this be API?

Also, maybe a more specific name is necessary. Or we consider that this class will be used everywhere someone needs an Elasticsearch-specific API, and in that case there should be a slightly more complex design (one central interface with multiple `Class<? extends XXX> getXXXAPI()` methods).